### PR TITLE
Feature/wallet login

### PR DESCRIPTION
**Pull request by jalarg**
# Summary
- Add initial NextJS project template
- Add Stellar icons
- Add login IU: Create wallet and generate keyPair
- Add commons: Button and Input components
- Add keypair and Handler to generate account Btn
- Add packages:  [1] Stellar-SDK - [2] antd [3] react icons
- Add navbar component
- Add cypress and set the initial configuration

# Details
- The home login page provides the user the options to:
 1) Click in the button "Generate key pair for new account" or "Confirm Wallet"
 2) If "Generate key pair for new for new account" is clicked, the user will be able to Confirm a new Secret and Private key or go back to main menu

# Evidence 

## Stellar wallet:
 
Wallet UI

wallet UI 2

### CODE REVIEW

File: src/app/actions/registerHandler.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Keypair } from "stellar-sdk";
+import setKeypair from "./setKeypair";
+import { ISetKeyPair } from "../types/types";
+
+function registerHandler({ setSecret, setPublicKey }: ISetKeyPair) {
+  // Call create random pair of keys
```
Comment (Genebson) [inline src/app/actions/registerHandler.ts]: If our code is legible and well-developed is not necessary to add comments. The code should speak for us, we already know by reading the pair variable that we're generating random keypairs.

File: src/app/actions/setKeypair.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Keypair } from "stellar-sdk";
+import { ISetKeyPair } from "../types/types";
+
+// Crear una nueva interfaz que extienda RegisterProps
```
Comment (Genebson) [inline src/app/actions/setKeypair.ts]: Spanglish, avoid this, please.

File: src/app/commons/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+const Button: React.FC<ButtonProps> = ({
+    type,
+    fullWidth,
+    children,
+    onClick,
+    secondary,
+    danger,
+    disabled,
+  }) => {
+    let buttonClasses = `flex justify-center rounded-md px-3 py-2 text-sm font-semibold focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2`;
```
Comment (Genebson) [inline src/app/commons/Button.tsx]: Why are we using backticks here? We're not using template string.

File: src/app/commons/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      buttonClasses += " text-white";
+    }
+  
+    if (danger) {
+      buttonClasses +=
+        " bg-rose-500 hover:bg-rose-600 focus-visible:outline-rose-600";
+    } else {
+      buttonClasses +=
+        " bg-blue-400 hover:bg-blue-500 focus-visible:outline-blue-500";
+    }
```
Comment (Genebson) [inline src/app/commons/Button.tsx]: How we can make this more dynamic?

File: src/app/commons/Input.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            placeholder={placeholder}
+            disabled={disabled}
+            value={value || ""}
+            readOnly={readOnly}
+            required={required}
+            className={`rounded-md px-3 py-2 font-semibold w-full text-gray-800 focus:ring-2 focus:ring-inset focus:ring-sky-600 sm:text-sm sm:leading-6`}
+          />
+          <CopyToClipboard
+            text={value || ""}
+            onCopy={() => alert("Texto copiado al portapapeles")}
```
Comment (Genebson) [inline src/app/commons/Input.tsx]: This should be written in English.

File: src/app/actions/setKeypair.ts (hunk 2 above)
Comment (jalarg) [reply src/app/actions/setKeypair.ts]: Issue addressed in commit 46392d0adecd8be079ef4945707510a632e02f81, which removes the unnecessary comments.

File: src/app/commons/Input.tsx (hunk 5 above)
Comment (jalarg) [reply src/app/commons/Input.tsx]: Issue addressed in commit 46392d0adecd8be079ef4945707510a632e02f81, which eliminates language mistakes.

File: src/app/actions/registerHandler.ts (hunk 1 above)
Comment (jalarg) [reply src/app/actions/registerHandler.ts]: Issue addressed in commit 46392d0adecd8be079ef4945707510a632e02f81, which removes unnecessary comments.

File: src/app/commons/Button.tsx (hunk 4 above)
Comment (jalarg) [reply src/app/commons/Button.tsx]: Issue addressed in commit 81c2ee1cc5605f1af059f2304ef79d6902f9374d, which refactors the button class name.

File: src/app/commons/Button.tsx (hunk 3 above)
Comment (jalarg) [reply src/app/commons/Button.tsx]: Issue addressed in commit 81c2ee1cc5605f1af059f2304ef79d6902f9374d, which refactors the button class name.

File: cypress/e2e/home.cy.ts [hunk 6]
```diff
@@ -0,0 +1,8 @@
+const ProjectURL = Cypress.env("BASE_URL");
```
Comment (Genebson) [inline cypress/e2e/home.cy.ts]: We declare variables in `camelCase`

File: src/app/commons/Button.tsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+  type,
+  fullWidth,
+  children,
+  onClick,
+  secondary,
+  danger,
+  disabled,
+}) => {
+  
+  let buttonClasses =
```
Comment (Genebson) [inline src/app/commons/Button.tsx]: I think this can be a constant variable. Thoughts?

File: src/app/components/Navbar.tsx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+import { Avatar } from "antd";
+import { FaRegMoon } from "react-icons/fa";
+import { FiSun } from "react-icons/fi";
+
+const Navbar = () => {
+  const [theme, setTheme] = useState<string | "light">("dark");
+
+  const handleChangeTheme = () => {
+    setTheme((prevTheme) => (prevTheme === "light" ? "dark" : "light"));
+  };
```
Comment (Genebson) [inline src/app/components/Navbar.tsx]:
```suggestion
const handleChangeTheme = () => setTheme((prevTheme) => (prevTheme === "light" ? "dark" : "light"));
```

File: src/app/commons/Button.tsx (hunk 7 above)
Comment (jalarg) [reply src/app/commons/Button.tsx]: I Agree , issue fixed in commit 645dc18b52e76988cc47099799ba8ffcb7515fc2.

File: cypress/e2e/home.cy.ts (hunk 6 above)
Comment (jalarg) [reply cypress/e2e/home.cy.ts]: Issue solved in commit 645dc18b52e76988cc47099799ba8ffcb7515fc2.

**Review by Genebson — APPROVED** [review]

**Pull request merged**